package.json

```
@@ -45,8 +45,10 @@
   "dependencies": {
     "@cosmjs/encoding": "0.23.1",
     "@cosmjs/math": "0.23.1",
+    "@cosmjs/proto-signing": "0.24.1",
     "@cosmjs/stargate": "0.25.0-alpha.2",
     "@cosmjs/tendermint-rpc": "0.25.0-alpha.2",
+    "@types/nock": "^11.1.0",
```

```suggestion
    "@types/nock": "^11.1.0",
```

This should be a dev dependecies?

---

This is true.  Fixed.

Thanks